### PR TITLE
It allows to obtain a snapshot draft

### PR DESCRIPTION
**Pull request by mirtagonz**
Add services to obtain and delete a snapshot draft
Include PR https://github.com/FundacionParaguaya/FP-PSP-SERVER/pull/116

### CODE REVIEW

File: src/main/java/py/org/fundacionparaguaya/pspserver/surveys/services/impl/SnapshotDraftServiceImpl.java [hunk 1]
```diff
@@ -110,23 +111,22 @@ public SnapshotDraft updateSnapshotDraft(Long id,
...
+                details.getUsername()).orElse(null);
+
+        if (user == null) {
+            return Collections.emptyList();
         }
 
+        ret = repository
+                  .findAll(where(userEquals(user.getId()))
+                  .and(likeFamilyName(familyName))
+                  .and(createdAtLess8Days()));
```
Comment (rodrigojv) [inline src/main/java/py/org/fundacionparaguaya/pspserver/surveys/services/impl/SnapshotDraftServiceImpl.java]: Can we have the number of days as a parameter?

File: src/main/java/py/org/fundacionparaguaya/pspserver/web/rest/SnapshotDraftController.java [hunk 2]
```diff
@@ -130,12 +131,13 @@ public SnapshotDraftController(SnapshotDraftService service) {
              code = 200,
              message = "The requested snapshot draft by user id",
              response = SurveyDefinition.class) })
-        public ResponseEntity<List<SnapshotDraft>> getSnapshotDraftByUser(                        
-                        @RequestParam(value = "description", required = true) 
+        public ResponseEntity<List<SnapshotDraft>> getSnapshotDraftByUser (
+                @RequestParam(value = "description", required = true)
                         String description,
```
Comment (rodrigojv) [inline src/main/java/py/org/fundacionparaguaya/pspserver/web/rest/SnapshotDraftController.java]: We can call this parameter `familyName` and keep the request param value as description.

File: src/main/java/py/org/fundacionparaguaya/pspserver/surveys/services/impl/SnapshotDraftServiceImpl.java [hunk 3]
```diff
@@ -110,23 +111,22 @@ public SnapshotDraft updateSnapshotDraft(Long id,
...
-            }
+
+        UserEntity user = userRepository.findOneByUsername(
+                details.getUsername()).orElse(null);
+
+        if (user == null) {
+            return Collections.emptyList();
         }
 
+        ret = repository
```
Comment (rodrigojv) [inline src/main/java/py/org/fundacionparaguaya/pspserver/surveys/services/impl/SnapshotDraftServiceImpl.java]: Remove the declaration above and instead do:
```java
List<SnapshotDraft> draftList = repo.findAll...
```

**Comment by mirtagonz** [comment]
The suggested changes were made! @rodrigojv

**Pull request merged**